### PR TITLE
A whole bunch of proofreading

### PR DESCRIPTION
**Pull request by InSimpleTermsJordan**
mostly punctuation and capitalization, but also a few spelling fixes and some rewording

Is it possible to make this link to the section?
### Gender Euphoria
Would it be worth linking to [the Wikipedia article for Plato's cave allegory](https://en.wikipedia.org/wiki/Allegory_of_the_cave)?

### Social Dysphoria
I'm assuming the forgotten adjective in "surprisingly phenomenon" was something like "common", so please correct it if it's wrong

Recommend rewording this, as it seems to imply that the discomfort women experience is what includes genuine attention rather than the discomfort AFAB trans people experience

strongly recommend rewording for redundancy reasons

do we really have to support Jeff Bezos?
### Am I Trans?
"it’s nearly impossible avoid listening to oneself over others." has been reworded as "it’s nearly impossible to avoid listening to others over oneself." as the former seems contradictory to the rest of the paragraph, but I may be missing something, so please correct me if it's wrong

I also suggest rewording "I thought being dysphoric all the time was normal human behavior" so as to not imply that dysphoria is a behavior and therefore a choice

Also could use confirmation that "If you want to be a girl and you’ve always thought of yourself as a guy, then you will probably be happier living as a girl." is worded correctly
### Clinical Diagnoses
Some of these changes are to bring the wording somewhat closer to the text of the DSM-5 without making it too binary-centric

Highly recommend a source on this (admittedly mostly because it interests me personally and I couldn't find anything about it when I looked it up)
### Estrogenic Second Puberty
Removed dead link to https://youtu.be/OROoZzoVwfk?t=12

### CODE REVIEW

**Comment by ErdaradunGaztea** [comment]
Regarding the second-to-last part, the one about PPV, you may look up r/salmacian subreddit for info (though I'm not sure if I'd use it as a source).

**Comment by InSimpleTermsJordan** [comment]
ok not sure how @gjvnq's commits got in here

**Comment by Twipped** [comment]
I think because you merged master instead of rebased. That tends to do that.

Regarding non-binary surgeries:
- QueerDoc:
  - [Phallus Preserving Vaginoplasty](https://queerdoc.com/phallus-preserving-vaginoplasty/) has a summary page on the subject
  - [Vagina Preserving Phalloplasty](https://queerdoc.com/vpp/)
- [Post Street Surgery Center](https://www.poststreetsurgery.com/gender-affirmation/non-binary/) (Thomas Satterwhite)
- Align Surgical Associates (also Satterwhite):
  - [Vagina Preserving Phalloplasty](https://www.alignsurgical.com/trans-men/vagina-preserving-phalloplasty/)
  - [Phallus Preserving Vaginoplasty](https://www.alignsurgical.com/non-binary/phallus-preserving-vaginoplasty/) (link seems to be broken at the moment)
  - [Non-Binary Surgeries at MoZaic Care](https://www.mozaiccare.net/gnc-nb-including-penile-preservatio) (Heidi Wittenberg)

> **do we really have to support Jeff Bezos?**

For many people, Amazon is the easiest and often cheapest means of getting ahold of goods, and [Mia herself](https://www.miaviolet.love/books/) links to the Amazon pages. But sure, we could add the [Book Depository](https://www.bookdepository.com/Yes--You-Are-Trans-Enough/9781785923159) link... oh wait, that's also owned by Amazon. Or we could add the link to [Waterstones](https://www.waterstones.com/book/yes-you-are-trans-enough/mia-violet/9781785923159), which is owned by the Elliot Investment Management hedge fund. Or we could link to [Barnes & Noble](https://www.barnesandnoble.com/w/yes-you-are-trans-enough-mia-violet/1127839104), which is also owned by Elliot Investment Management.

[Well damn...](https://www.linkedin.com/pulse/amazon-right-us-elliott-left-kieron-smith/?trk=public_profile_article_view)

**Comment by InSimpleTermsJordan** [comment]
> > **do we really have to support Jeff Bezos?**
> 
> For many people, Amazon is the easiest and often cheapest means of getting ahold of goods, and [Mia herself](https://www.miaviolet.love/books/) links to the Amazon pages. But sure, we could add the [Book Depository](https://www.bookdepository.com/Yes--You-Are-Trans-Enough/9781785923159) link... oh wait, that's also owned by Amazon. Or we could add the link to [Waterstones](https://www.waterstones.com/book/yes-you-are-trans-enough/mia-violet/9781785923159), which is owned by the Elliot Investment Management hedge fund. Or we could link to [Barnes & Noble](https://www.barnesandnoble.com/w/yes-you-are-trans-enough-mia-violet/1127839104), which is also owned by Elliot Investment Management.

Looked into it, and it seems it is possible to link directly to [the book's page on the site of the publisher](https://us.jkp.com/products/yes-you-are-trans-enough), [Jessica Kingsley Publishers](https://en.wikipedia.org/wiki/Jessica_Kingsley_Publishers) (and I should probably clarify that I'm not affiliated with them despite my initials also being "JKP" lol), which is controlled by Hachette, which is controlled by Lagardère Publishing, a subsidiary of Lagardère Group.
I'm not particularly fond of Hachette due to [its attacks on the Internet Archive](https://www.thenation.com/article/society/publishers-are-taking-the-internet-to-court/), but it's probably still better than the other two. The book is also available on a certain shadow library, though it's probably not advisable to link to its page there or to the book's IPFS link.
